stable/confidant/values.yaml

```diff
@@ -7,14 +7,16 @@ awsDefaultRegion: "us-east-1"
 
 image:
   repository: "lyft/confidant"
-  tag: "1.9.0"
+  tag: "latest"
```

Comment:
I really, really dislike putting `latest` in as a default value tag...

Reply:
Only did it because it's a public chart.